package/MDAnalysis/analysis/rms.py

```diff
@@ -333,10 +328,12 @@ def __init__(self, atomgroup, reference=None, select='all',
             .. Note:: Experimental feature. Only limited error checking
                       implemented.
 
-        filename : str (optional)
+        filename : str, optional
             write RSMD into file file :meth:`RMSD.save`
-        mass_weighted : bool (optional)
+        mass_weighted : bool (deprecated)
              do a mass-weighted RMSD fit
```

Comment:
There should also be some markup at the bottom such as 
```reST
.. versionchanged:: 0.16.0
   Flexible weighting scheme with new ``weights`` keyword.

.. deprecated:: 0.16.0
   Instead of ``mass_weighted=True`` use new ``weights='mass'`` 
```